src/main/java/org/aksw/gerbil/datatypes/ExperimentType.java

```diff
@@ -285,11 +321,22 @@ public boolean equalsOrContainsType(ExperimentType type) {
         }
         case OKE2018Task4:{
         	switch(type) {
+            case OKE2018Task4:
         	case RE:
-        	case OKE2018Task4:
+        	case KE:
         	case ERec:
         		return true;
         	}
```

Comment:
As already discussed before, at the end of OKE2018Task4 either `default` or all the other expeirment tasks are missing which should return false. Otherwise, these cases will simply "fall through" into the KE case following below.